py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/utils.py

```diff
@@ -0,0 +1,28 @@
+import os
```

Comment:
we're already using this in performance manager. not sure what the best place for it is since its useful in multiple places.